src/legacy/core_plugins/vis_type_vislib/public/vislib/components/legend/legend.tsx

```diff
@@ -22,14 +22,24 @@ import { compact, uniq, map } from 'lodash';
 
 import { i18n } from '@kbn/i18n';
 import { EuiPopoverProps, EuiIcon, keyCodes, htmlIdGenerator } from '@elastic/eui';
+import { IAggConfig } from 'ui/agg_types';
 
 // @ts-ignore
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { createFiltersFromEvent } from '../../../../../data/public/actions/filters/create_filters_from_event';
 import { CUSTOM_LEGEND_VIS_TYPES, LegendItem } from './models';
 import { VisLegendItem } from './legend_item';
 import { getPieNames } from './pie_utils';
-import { getTableAggs } from '../../../legacy_imports';
+
+import { Vis } from '../../../../../visualizations/public';
+import { tabifyGetColumns } from '../../../../../../ui/public/agg_response/tabify/_get_columns';
```

Comment:
There should be no legacy imports within the vis types, please re-export those via `legacy_imports`. I'm currently working on a separate PR that will break on these changes.